resources/documentation/stories.rst

```diff
@@ -0,0 +1,467 @@
+=======
+Stories
+=======
+
+This part contains few stories to help you face common tasks / issues seen
+while using the stack.
+
+Management and main configuration
+=================================
+
+Versioning the inventory
+------------------------
+
+It may be a good idea to version the whole /etc/bluebanquise/inventory/
+folder, to keep track of all changes made, and be able to revert in case of
+issues.
+
+.. note::
+  If used, it may be also a good idea to version the
+  /etc/bluebanquise/playbooks/ and /etc/bluebanquise/roles/custom/ folders.
+
+It may even be better to use an external to the cluster git platform (gitlab,
+etc.) to backup the whole configuration, and be able to push changes from the
+cluster to this platform.
+
+To create a basic git repository, assuming you already have an in place
+configuration inside /etc/bluebanquise/inventory/, do:
+
+.. code-block:: text
+
+  cd /etc/bluebanquise/inventory/
+  git init
+  git add --all
+  git commit --author="my_name <my_name@my_mail.org>" -m "First commit"
```

Comment:
I would recommend:
```
git config user.name "Firstname Lastname"
git config user.email me@mydomain.org
```
before commit, then you can skip --author.

Reply:
Idea is: if there are multiple admins using the same git. Using the --author, they can work on the same repository, knowing who do what.

Reply:
That's what I thought, however on some setup git will force to define user.name and user.email.
I have 2 ideas in mind to solve this multiple admin case. Let's merge like this, I will propose something later.

Reply:
Ok for me. We will have to add more content in stories anyways. 🙂